### ISSUE TITLE
"str_replace() can be used instead" false positive when a limit has been set

### ISSUE THREAD

**Issue by michielroding** [kalessil/phpinspectionsea#401]
```php
preg_replace('/STRING/', 'OTHER', 'I got a STRING in my STRING', 1);
```

Due to the limit set to 1, a str_replace() will not behave the same way.

**Comment by kalessil** [comment]
Ouch, how I could miss that. Fixed.
